example/js/api/diplomat-runtime.mjs

```diff
@@ -182,15 +182,102 @@ export class DiplomatBuf {
 	}
 
 	constructor(ptr, size, free) {
-	this.ptr = ptr;
-	this.size = size;
-	// Generated code calls one of methods these for each allocation, to either
-	// free directly after the FFI call, to leak (to create a &'static), or to
-	// register the buffer with the garbage collector (to create a &'a).
-	this.free = free;
-	this.leak = () => { };
-	this.garbageCollect = () => DiplomatBufferFinalizer.register(this, this.free);
+		this.ptr = ptr;
+		this.size = size;
+		// Generated code calls one of methods these for each allocation, to either
+		// free directly after the FFI call, to leak (to create a &'static), or to
+		// register the buffer with the garbage collector (to create a &'a).
+		this.free = free;
```

Comment:
thought: I didn't realize we were tab-indenting in this fine. not a huge deal, does become one of the few places we tab-indent in this repo

Reply:
I think this might be a holdover from the original JS backend, I've opened #601 accordingly.